BosTau.py

```diff
@@ -0,0 +1,211 @@
+"""
+Catalog definitions for Bos Taurus
+"""
+import math
+import logging
+
+import msprime
+
+import stdpopsim
+
+logger = logging.getLogger(__name__)
+
+###########################################################
+#
+# Genome definition
+#
+###########################################################
+
+
+# Create a string of chromosome lengths for easy parsing
+_chromosome_data = """\
+chr1    158534110
```

Comment:
Could you add a comment here about which reference assembly (and version) the chromosome lengths come from?

Reply:
If you have a URL for where you got it from that would be nice to have in a comment as well

Reply:
The reference assembly comes from Ensembl genome browser... Genome assembly: ARS-UCD1.2 (GCA_002263795.2) .  https://www.ensembl.org/Bos_taurus/Info/Annotation
Worked out the chromosome lengths from each "chromosome summary" in Ensembl. Copied and pasted.
Eg. for Chr1 ....https://www.ensembl.org/Bos_taurus/Location/Chromosome?r=1:421347-158534110
Chr2... https://www.ensembl.org/Bos_taurus/Location/Chromosome?r=2:14572483-14672483